salt/engines/kujira_parser/kujira_event_parser.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
Remove directory kujira_parser.
Change name 'kujira_event_parser.py' to event_parser.py